docs/admin/accessing-the-api.md

```diff
@@ -53,7 +53,9 @@ If the request cannot be authenticated, it is rejected with HTTP status code 401
 Otherwise, the user is authenticated as a specific `username`, and the user name
 is available to subsequent steps to use in their decisions.  Some authenticators
 may also provide the group memberships of the user, while other authenticators
-do not (and expect the authorizer to determine these).
+do not; an authorizer may determine as well as use groups (or any other
+auxiliary concepts), as a private detail of that authorizer (none of the standard
+authorizers determines groups).
```

Comment:
The sentence is a little long and could stand a clarity edit. How about:

"Some authenticators may also provide a user's group memberships, while other authenticators might not; the latter expect the authorizer to determine a user's group membership. Note that none of the standard authorizers determine groups, but an authorizer can theoretically determine or use groups as a private detail."